elastic/integrations
# [GCP VertexAI] Add support for different deployment types

- Enhancement

This PR adds support for provisioned throughput metrics monitoring. This is useful for the PT deployments.

Add request_type control filter in the dashboard to filter data based on the deployment type either dedicated or shared.

Update documentation to add information related to the different deployment types and update prediction error count chart group_by top n values consistent with other chart.

## Checklist


## Author's Checklist


## How to test this PR locally

- Install the integration and verify the metrics collection happens for both the deployment type.


## Related issues

- Closes [PT deployment monitoring](https://github.com/elastic/obs-integration-team/issues/102)

## Screenshots

<img width="1725" alt="gcp-vertexai-overview-dashboard" src="https://github.com/user-attachments/assets/c50c5f75-2629-4418-9430-fd088e7c050e">
